### PR TITLE
Create publish-1.1-docs.yml

### PR DESCRIPTION
**Pull request by pjfanning**
CI job to publish docs

### CODE REVIEW

**Comment by Roiocam** [comment]
@pjfanning I have been rebased to the main branch, let us see what happens that CI shows.

File: .github/workflows/publish-1.1-docs.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      - name: Create directory structure
+        run: |-
+          mkdir -p target/nightly-docs/docs/pekko-persistence-jdbc/1.1.0-M1/
+          mkdir -p target/nightly-docs/docs/pekko-persistence-jdbc/1.1/
+          cp -r docs/target/paradox/site/main/ target/nightly-docs/docs/pekko-persistence-jdbc/1.1.0-M1/docs
+          cp -r docs/target/paradox/site/main/ target/nightly-docs/docs/pekko-persistence-jdbc/1.1/docs
+          rm -r docs/target/paradox/site/main/
+          cp -r target/scala-2.13/unidoc target/nightly-docs/docs/pekko-persistence-jdbc/1.1.0-M1/api
+          cp -r target/scala-2.13/unidoc target/nightly-docs/docs/pekko-persistence-jdbc/1.1/api
+          rm -r target/scala-2.13/unidoc
```
Comment (Roiocam) [inline .github/workflows/publish-1.1-docs.yml:45]: Why we didn't publish japi docs?

**Review by nvollmar — APPROVED** [review]
lgtm

File: .github/workflows/publish-1.1-docs.yml (hunk 1 above)
Comment (pjfanning) [reply .github/workflows/publish-1.1-docs.yml:45]: We only publish Java APIs for core and HTTP modules.

https://pekko.apache.org/modules.html

You can raise an issue but if you check Akka docs, it is basically the same for Akka (that specific Javadocs as opposed to Scaladocs are not published for all modules).

Comment (Roiocam) [reply .github/workflows/publish-1.1-docs.yml:45]: It seems fair to me, I just asking.

**Review by Roiocam — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**